test/update-hlds-cstrike.env

```diff
@@ -10,5 +10,6 @@ APPID=90
 GAME=cstrike
 GAME_UPDATE_COUNT=1
 NO_TEST=
+NO_PULL=true
```

Comment:
Also `NO_PULL` should come before `NO_TEST`.

Reply:
The PR looks generally good to me. I've fixed the positions of the switch: https://github.com/joeltimothyoh/docker-sourceservers/pull/14/commits/9069ac01c8eb434e451343c472067ecf2be4cc1e @leojonathanoh